### PR TITLE
Added file Software_Architecture.md

### PR DESCRIPTION
**Pull request by NoorMohamed1885**

### CODE REVIEW

File: decision_support/Software_Architecture.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Software Architecture Documentation
+## [User] ->
+```User which is the person accessing the product.```
+## [Ubuntu Linux Virtual Machine] -> 
+```Have the product open in something like a virtual machine.```
+## [Command Line] ->
+```The command line which allows you to install applications(OpenJDK and Apache Ant).```
+## [OpenJDK]
+```Have OpenJDK which allows you to create and run java files. Source files location "cs3250_practice/heloCS3250/src/metrocs"```
```
Comment (Garret-Gallant) [inline decision_support/Software_Architecture.md]: Typo on "cs3250_practice/heloCS3250/src/metrocs"

cs3250_practice/helloCS3250/src/metrocs

Comment (NoorMohamed1885) [reply decision_support/Software_Architecture.md]: Thanks file should now be updated with changes.

File: decision_support/Software_Architecture.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Software Architecture Documentation
+## [User] ->
+```User which is the person accessing the product.```
+## [Ubuntu Linux Virtual Machine] -> 
+```Have the product open in something like a virtual machine.```
```
Comment (Garret-Gallant) [inline decision_support/Software_Architecture.md]: Add version of ubuntu that is supported by the VM, add options for VM software on windows/macx86/mac (Apple Silicon) machines? 

https://www.virtualbox.org/
https://mac.getutm.app/

**Review by Garret-Gallant — APPROVED** [review]
Other than the comments added, this looks good! 🚀 Nice work!

File: decision_support/Software_Architecture.md (hunk 2 above)
Comment (NoorMohamed1885) [reply decision_support/Software_Architecture.md]: Thanks file should now be updated with changes.

**Pull request merged**